src/views/components/cart.js

```diff
@@ -111,14 +98,15 @@ export default class Cart extends Component {
                           }}>
                           <Text style={styles.qtyChanger}> + </Text>
                         </TouchableOpacity>
-                        <Text style={{fontSize: 15}}> {item.quantity} </Text>
+                        <Text style={styles.decrQty}> {item.quantity} </Text>
                         <TouchableOpacity
                           onPress={() => {
                             if (item.quantity > 1) {
                               item.quantity--;
                               this.setState({dataList: this.state.dataList});
 
                               this.setState({
```

Comment:
You are still using multiple setStates inside the same function. try merging them in one single setState